### PR TITLE
feat: add sortedSetGetScore

### PR DESCRIPTION
**Pull request by malandis**
Adds the sortedSetGetScore method, response type, and tests to the
client.

### CODE REVIEW

File: src/Cache/CacheClient.php [hunk 1]
```diff
@@ -1841,6 +1842,57 @@ public function sortedSetFetchByRank(string $cacheName, string $sortedSetName, ?
...
+     * @param string $cacheName Name of the cache that contains the sorted set.
+     * @param string $sortedSetName The sorted set to fetch the score from.
+     * @param string $value The value to fetch the score for.
+     * @return SortedSetGetScoreResponse Represents the result of the sorted set get score operation. This result is resolved to a type-safe object of one of the following types:<br>
+     * * SortedSetGetScoreHit<br>
+     * * SortedSetGetScoreMiss<br>
+     * * SortedSetGetScoreError<br>
+     * Pattern matching can be to operate on the appropriate subtype:<br>
+     * <code>
+     *     if ($hit = $response->asHit()) {
```
Comment (rishtigupta) [inline src/Cache/CacheClient.php:1884]: Can we add the handling "miss" response type case here?

File: src/Cache/CacheClient.php [hunk 2]
```diff
@@ -1841,6 +1842,57 @@ public function sortedSetFetchByRank(string $cacheName, string $sortedSetName, ?
...
+     * @param string $value The value to fetch the score for.
+     * @return ResponseFuture<SortedSetGetScoreResponse> A waitable future which will provide the result of the sorted set get score operation upon a blocking call to wait:<br />
+     * <code>$response = $responseFuture->wait();</code><br />
+     * The response represents the result of the sorted set get score operation. This result is resolved to a type-safe object of one of the following types:<br>
+     * * SortedSetGetScoreHit<br>
+     * * SortedSetGetScoreMiss<br>
+     * * SortedSetGetScoreError<br>
+     * Pattern matching can be to operate on the appropriate subtype:<br>
+     * <code>
+     *     if ($hit = $response->asHit()) {
```
Comment (rishtigupta) [inline src/Cache/CacheClient.php:1859]: Can we add the handling "miss" response type case here?

**Review by rishtigupta — APPROVED** [review]
looks good to me, just a few nits (can be handled separately if you like)

File: src/Cache/CacheClient.php (hunk 1 above)
Comment (malandis) [reply src/Cache/CacheClient.php]: Done in [123cf56](https://github.com/momentohq/client-sdk-php/pull/221/commits/123cf56528e635c20030822498747ea0e1ee0729)

File: src/Cache/CacheClient.php (hunk 2 above)
Comment (malandis) [reply src/Cache/CacheClient.php]: Done in [123cf56](https://github.com/momentohq/client-sdk-php/pull/221/commits/123cf56528e635c20030822498747ea0e1ee0729)

**Review by rishtigupta — APPROVED** [review]
:shipit:

**Pull request merged**